packages/js/components/src/section/section.tsx

```diff
@@ -27,13 +31,16 @@ export function Section( { component, children, ...props } ) {
 					{ component === false ? (
 						children
 					) : (
+						// @ts-expect-error: Raises JSX component `Component` doesn't have constructor to call error.
+						// Suppressing the error since this component accepts func, string, and bool
+						// I wasn't able to find to correct type.
 						<Component { ...props }>{ children }</Component>
 					) }
 				</Level.Provider>
 			) }
 		</Level.Consumer>
 	);
-}
+};
 
 Section.propTypes = {
```

Comment:
@moon0326 As @rjchow's [suggestion](https://github.com/woocommerce/woocommerce/pull/36265#discussion_r1061397784), we can remove the propTypes now (also in your other PRs). 🙂 

I think it's easier to maintain the TypeScript components without propTypes, though they seem to [serve different purposes](https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application).

Reply:
Sounds good 👍

Reply:
@chihsuan Updated in https://github.com/woocommerce/woocommerce/pull/36298/commits/21e463cc4f57ccf8aa5a679f49b343c06baa2c7e